src/blurredLocation.js

```diff
@@ -152,6 +153,25 @@ BlurredLocation = function BlurredLocation(options) {
     return getMinimumGridWidth(options.pixels).precision;
   }
 
+  function setBlurred(boolean) {
+      if(boolean && !blurred) {
+        gridSystem.addGrid();
+        blurred = true;
+      }
+      else if(!boolean) {
+        blurred = false;
+        gridSystem.removeGrid();
+      }
+  }
+
+  function isBlurred() {
+    return blurred;
+  }
+
+  function obscureLocation() {
+    setBlurred(document.getElementById("obscureLocation").checked);
```

Comment:
OK but this is UI code so lets merge this into the new UI module -- merge that in first.